### PR TITLE
Add links to WikiWatershed content site.

### PR DESCRIPTION
**Pull request by caseypt**
* Per instructions from Stroud.

Connects #1210

**Testing instructions**
- Verify all of the links are valid and in the correct place (see #1210).

image

image

image

image

### CODE REVIEW

File: src/mmw/apps/water_balance/templates/home_page/index.html [hunk 1]
```diff
@@ -331,6 +332,10 @@ <h1 class="infil">0.0 cm</h1>
                 </ul> <!-- End Soil Thumbs-->
             </div> <!-- End Soils -->
 
+            <div class="row pad-sm">
+                <p class="info text-muted">Explore how land use and soil determine runoff with our <a target="_blank" href="https://app.wikiwatershed.org/micro/">Micro Site Storm Model</a>. Info and help at <a target="_blank" href="http://wikiwatershed.org/model/">http://wikiwatershed.org/model/</a>.</p>
```
Comment (mmcfarland) [inline src/mmw/apps/water_balance/templates/home_page/index.html]: This chunk got the wrong text.  According to the issue, it should be:

> Apply our Site Storm Model to your watershed. (link to https://app.wikiwatershed.org/). Info and help at http://wikiwatershed.org/model/.

Comment (caseypt) [inline src/mmw/apps/water_balance/templates/home_page/index.html]: Good catch. I'll update this.

**Comment by caseypt** [comment]
Updated:

image

**Comment by mmcfarland** [comment]
+1, looks good.

**Pull request merged**